### PR TITLE
[Enhancement](Nereids)add single sql fall back to original planner hint

### PR DESCRIPTION
**Pull request by sohardforaname**
# Proposed changes

Issue Number: close #xxx

## Problem summary

now we can use /*+ SET_VAR(enable_nereids_planner="false") */ to disable nereids in a single sql.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [x] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.77 seconds
 stream load tsv:          478 seconds loaded 74807831229 Bytes, about 149 MB/s
 stream load json:         25 seconds loaded 2358488459 Bytes, about 89 MB/s
 stream load orc:          76 seconds loaded 1101869774 Bytes, about 13 MB/s
 stream load parquet:          33 seconds loaded 861443392 Bytes, about 24 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230322000904_clickbench_pr_118233.html

**Review by Kikyou1997 — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/nereids/processor/pre/EliminateLogicalSelectHint.java [hunk 1]
```diff
@@ -72,5 +72,19 @@ private void setVar(SelectHint selectHint, StatementContext context) {
                 }
             }
         }
+        // if sv set enable_nereids_planner=true and hint set enable_nereids_planner=false, we should set
+        // enable_fallback_to_original_planner=true and revert it after executing.
+        // throw exception to fall back to original planner
+        if (!sessionVariable.isEnableNereidsPlanner()) {
+            String key = "enable_fallback_to_original_planner";
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/processor/pre/EliminateLogicalSelectHint.java:79]: we should use `org.apache.doris.qe.SessionVariable#ENABLE_FALLBACK_TO_ORIGINAL_PLANNER`

**Comment by sohardforaname** [comment]
run buildall

**Review by englefly — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**